src/Infrastructure/Event/CliRunnerSetupListener.php

```diff
@@ -7,19 +7,19 @@
 
 namespace SprykerSdk\Sdk\Infrastructure\Event;
 
-use SprykerSdk\Sdk\Core\Application\Dependency\CliCommandRunnerInterface;
+use SprykerSdk\Sdk\Infrastructure\Command\CliCommandRunnerInterface;
```

Comment:
All the Event stuff should be moved to the Core/Application layer. It doesn't contain any concrete infra logic. Completely application layer logic.

Reply:
there're no logic. it connect data from other layers